build.gradle.kts

```
@@ -45,7 +45,7 @@ idea {
 plugins {
     idea
     kotlin("jvm") version "1.7.21"
-    id("org.jetbrains.intellij") version "1.11.0"
+    id("org.jetbrains.intellij") version "1.13.2"
```

```suggestion
    id("org.jetbrains.intellij") version "1.13.3"
```
we can bump to the very last version  ;) (released 2 day ago )